assets/js/theme/common/utils/banner-utils.js

```diff
@@ -0,0 +1,27 @@
+import { isBrowserIE } from './ie-helpers';
+
+const bannerUtils = {
+    dispatchProductBannerEvent: (productAttributes) => {
+        let price = 0;
+
+        if (!productAttributes.price || isBrowserIE) return;
```

Comment:
Don't you think it makes sense to switch these lines order - move if statement on line5?  if it's true we don't need to do extra action & create variable.